jenkins-pipelines/features-sla-100perc-throughput_latency_cache_load.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
please remove all the 100perc from the file names.
change the file name to features-sla-read-throughput-vs-latency-cache-only.

Change test name accordingly (test_read_throughput_vs_latency_cache_only)